dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/utils/SwitchTaskUtils.java

```diff
@@ -50,6 +50,10 @@ public class SwitchTaskUtils {
             "eval",
             "function",
             "import",
+            ".",
+            "()",
+            "[",
+            "]",
```

Comment:
can we get the final exec command in user input? such as we can convert below code

```js
var a = Java.type("ja" + "va.lang.Runtime");
var b = a.getRuntime();
b.exec(${cmd})
```

to 

```js
var a = Java.type("ja" + "va.lang.Runtime");
var b = a.getRuntime();
Java.type("ja" + "va.lang.Runtime").getRuntime().exec(${cmd})
```

If we can do it, I think that would be easier to check

Reply:
It's hard to convert because the code are executed in js engine.